core/runtime/impl/wasm_memory_impl.cpp

```diff
@@ -6,38 +6,23 @@
 #include "runtime/impl/wasm_memory_impl.hpp"
 
 namespace kagome::runtime {
-
-  WasmMemoryImpl::WasmMemoryImpl()
-      : offset_{1}  // We should allocate very first byte to prohibit allocating
+  WasmMemoryImpl::WasmMemoryImpl(wasm::ShellExternalInterface::Memory *memory)
+      : memory_(memory),
+        offset_(1)  // We should allocate very first byte to prohibit allocating
```

Comment:
Why did it change to ()?

Reply:
it was removed and then returned. No problem, I will return {}